### DISCUSSION TITLE
How to register an extension to an IPPBX behind LibreSBC

### DISCUSSION THREAD

**Discussion by nantoSur** [hnimminh/libresbc#177] [General]
User (Public IP) - Registering Extension 1000
       |
       v
LibreSBC (Public IP XX.XX.XX.XX)
       |
       v
LibreSBC (Private IP 192.168.122.253)
       |
       v
IPPBX (Private IP 192.168.122.200, Port 5080, Extension 1000)

1. Can LibreSBC function as a proxy to handle SIP registration from external devices?
2. How does LibreSBC forward SIP REGISTER messages from external users to the internal IPPBX?

**Comment by git-root** [comment]
@nantoSur Found any info regarding this query? I am looking for similar usecase as well

**Comment by hnimminh** [comment]
Hi @nantoSur @git-root thanks for interesting in LibreSBC, 
1. LibreSBC as an registrar to offload registrar traffic for core server.
2. Since above LibreSBC does not forward, it consume and handle SIP REGISTER. you will need to config routing to route call from your PBX to your extension & vice-versa
